doc/changes.rst

```diff
@@ -7,7 +7,6 @@ Change Log
 
 *
 
-
```

Comment:
I don't see how removing one line addresses my concern about not trying to set up for a tag in a merge.